app/models/setting/puppet.rb

```diff
@@ -1,35 +1,7 @@
+# Do not use - these settings are now in Setting::Configuration, but this model
+# has to stay in order to be able to migrate them
```

Comment:
Use a Fake* type class in the migration, or just use SQL if possible, instead of keeping an old class around.